tests/t_old/test-create-delete-key

```diff
@@ -0,0 +1,51 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
`t_old` seems like something you have been doing locally to test, it should be removed from this PR.